internal/app/machined/pkg/runtime/v1alpha1/platform/openstack/openstack_test.go

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-package openstack_test
+package openstack //nolint:testpackage
```

Comment:
why the rename here, it should be a test package

Reply:
It is almost everywhere here. I need it to check private functions/variables.